modules/customization/proc-loading-custom-theme-using-dynamic-plugin.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// assembly-customize-rhdh-theme.adoc
+
+[id="proc-loading-custom-theme-using-dynamic-plugin-_{context}"]
+= Loading a custom {product-short} theme by using a dynamic plugin
+
+You can load a custom {product-short} theme from a dynamic plugin.
+
+.Procedure
+
+. Export a theme provider function in your dynamic plugin, for example:
++
+[source,javascript]
+----
+import { lightTheme } from './lightTheme'; // some custom theme
+import { UnifiedThemeProvider } from '@backstage/theme';
+export const lightThemeProvider = ({ children }: { children: ReactNode }) => (
+  <UnifiedThemeProvider theme={lightTheme} children={children} />
+);
+----
```

Comment:
Maybe after this section would be a good place to include a link to a more [complete upstream example](https://backstage.io/docs/getting-started/app-custom-theme#example-of-a-custom-theme).  We just have to be careful that by this point the user understands that the method of incorporating a theme into RHDH is via the dynamic plugin mechanism.

Reply:
@gashcrumb We've linked to the [Creating a Custom Theme](https://backstage.io/docs/getting-started/app-custom-theme/#creating-a-custom-theme) section, is this ok or should we just link to the [Example of a custom theme](https://backstage.io/docs/getting-started/app-custom-theme/#example-of-a-custom-theme)

Reply:
@gashcrumb Does this make sense ^^^